### PR TITLE
feat: add connected wallet menu

### PR DESCRIPTION
**Pull request by MonikaCat**
## Description

Closes: #XXXX
[BDU-670](https://forbole.atlassian.net/browse/BDU-670)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] ran linting
- [ ] wrote tests where necessary
- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] targeted the correct branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [x] added an entry to the `CHANGELOG.md` file

### CODE REVIEW

File: apps/web/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Unreleased
 
+## Changes
+
 - Display `requested amount` and `recipient` fields inside Community Pool Spend proposal details page ([\#1053](https://github.com/forbole/big-dipper-2.0-cosmos/pull/1053))
+- Added Connect Wallet pop up screen ([\#1068](https://github.com/forbole/big-dipper-2.0-cosmos/pull/1068))
```
Comment (teamchong) [inline apps/web/CHANGELOG.md:6]: Hi @MonikaCat, I propose to use `changesets` to handle CHANGE LOG from now on

https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

for example, the following change will be created.
https://app.warp.dev/block/N26x633kwhHnNUWQbqEVnx
https://app.warp.dev/block/irqexovjFIzakX0xvgnNkF
and then the changes will be committed
when we merge change into `main`
https://github.com/forbole/big-dipper-2.0-cosmos/blob/88607098d57652cf12a12f7f4704f557c4a2e77d/.github/workflows/release.yml#L30
we have a GItHub action to append change to CHANGE.md

What do you think?

**Review by teamchong — COMMENTED** [review]
utACK

File: apps/web/CHANGELOG.md (hunk 1 above)
Comment (MonikaCat) [reply apps/web/CHANGELOG.md:6]: Ok nice! I've just updated it, can you review it again pls? Thanks :D

Comment (rachelhox) [reply apps/web/CHANGELOG.md:6]: @teamchong does it apply to my PR #1085 as well ? Whenever there's a feature to be added, I'd add the commit in CHANGE LOG inside the `.changeset/` ?

Comment (teamchong) [reply apps/web/CHANGELOG.md:6]: @rachelhox no only use changeset for big-dipper repo for now

**Comment by MonikaCat** [comment]
Closing as this is now replaced with https://github.com/forbole/big-dipper-2.0-cosmos/pull/1128

**Pull request closed without merging**